explainer.md

```diff
@@ -0,0 +1,59 @@
+# Server Timing Explained
+
+## What's All This About?
+Server Timing allows developers to monitor the performance of server-handling of requests of every resource that makes up a page by overloading Navigation (for the base HTML) and Resource Timing (for scripts, images, etc). This new API enables developers to have insight into performance bottlenecks of the `Request` block below, that is, after `requestStart` and before `responseStart`.
```

Comment:
Nitpicking language, but perhaps..

"Server Timing allows developers to monitor the performance of server-handling of requests of every resource that makes up a page by extending Navigation Timing and Resource Timing APIs. The new attributes exposed by Server Timing enable developers to have insights.."